.gitignore

```diff
@@ -1 +1,5 @@
 *~
+build.ad-server
+build.client
+build.server
+build.toolbox
```

Comment:
I think you missed the `.`(dot) in the beginning of each entry. Also we have 2 more of such files related to nightly builds: _.build.nightly-server_ and _.build.nightly-ad-server_

Reply:
> I think you missed the `.`(dot) in the beginning of each entry. Also we have 2 more of such files related to nightly builds: _.build.nightly-server_ and _.build.nightly-ad-server_

All fixed now, thanks!